aptos-move/aptos-vm/src/move_vm_ext/vm.rs

```diff
@@ -104,16 +104,18 @@ impl Deref for MoveVmExt {
     }
 }
 
-pub fn verifier_config(treat_friend_as_private: bool) -> VerifierConfig {
+pub fn verifier_config(_treat_friend_as_private: bool) -> VerifierConfig {
```

Comment:
let's delete this in the calls?

Reply:
I'll remove this in a separate PR to make the changes minimal here. Guess we will need to cherrypick this into production branch.